app/controllers/bookmarked_posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class BookmarkedPostsController < ApplicationController
+  def index
+    @bookmark_ids = BookmarkedPost.where(user: current_user)
```

Comment:
Šeit tev vajadzetu izmantot vnk @posts instances mainigo, lai būtu skaidraks un tad atlasit visu ar SQL(surprise!).
https://guides.rubyonrails.org/active_record_querying.html

Ja tev visas asociacijas ir pareizas, tad būs tā:
`@posts = Post.joins(:bookmarked_posts).where(bookmarked_posts: { user: current_user })`

Reply:
Radija ka kkada kluda ar to `bookmarked_posts` , so es mazliet pielagoju kodu lai isaks butu.